### ISSUE TITLE
Different ContextMenu for children

### ISSUE THREAD

**Issue by rhoman** [vkbansal/react-contextmenu#23]
Hello,

Is this possible to define contextMenu for whole app container and different contextMenus for some of it's childrens?

For example, let's say I have structure like this (simplified to HTML):

```
<div class="container">
  <ul class="list1">
    <li></li>
  </ul>
  <ul class="list2">
    <li></li>
  </ul>
<div>
```

and I'd like to have contextMenu for whole .container and different contextMenus if you click on any of .list elments.

Right now when I've defined different menus for both lists it works right form the box.
After I've added menu for container I get it even when I click on list elements.

Best,
Martin

**Comment by vkbansal** [comment]
https://vkbansal.github.io/react-contextmenu/#/multiple-menus

see examples folder for implementation

**Comment by rhoman** [comment]
I've follow the example and it works great for siblings. But When i try to use them for parent->children elements, it always displays parent menu (even when I click on children).

**Comment by vkbansal** [comment]
Have you used different identifiers for the menus?

**Comment by rhoman** [comment]
Yes I did

**Comment by rhoman** [comment]
Could you please add demo with this example?

**Comment by vkbansal** [comment]
I'm a bit occupied until weekend. Will have a look then

On 14 July 2016 at 16:58, rhoman <notifications@github.com> wrote:

> Could you please add demo with this example?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/vkbansal/react-contextmenu/issues/23#issuecomment-232639157>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFp9b-1TRRy_HL4q2kkv7bZMKhOaAploks5qVh1mgaJpZM4JLXp9>
> .
>



-- 
Regards,
Vivek Kumar Bansal.

**Comment by rhoman** [comment]
Hi,

did you manage to take a look on this? :)

**Comment by vkbansal** [comment]
Sorry I couldn't. I'm very busy with my day job.

**Comment by vkbansal** [comment]
I've released `v2.0.0-alpha.1`. Do check it out. Any feedback is welcome 😃
